src/components/card-link/types.ts

```diff
@@ -1,3 +1,4 @@
+import { MaybeInternalLinkProps } from 'components/maybe-internal-link/types'
 import { ReactNode } from 'react'
```

Comment:
Two things here:

- Could you move the `'react'` import to be first?
- Could you update `MaybeInternalLink` to export the type from the component file to shorten this import statement?